.ruby-version

```diff
@@ -1 +1 @@
-2.1.2
+2.3.8
```

Comment:
@thermistor what big issues should we be looking for in this jump in Ruby version? Scrubbing code for particular methods that have significant changes?

Reply:
I can't see any reading the ruby release blog about the updates.
Biggest impact is we could start using the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) `&.` instead of using `.try`